src/lambdaisland/glogc.cljc

```diff
@@ -51,9 +51,13 @@
        (target :clj  (#'pedestal/log-expr &form :error keyvals)
                :cljs (#'glogi/log-expr &form :error keyvals)))
 
-     (defmacro spy [expr]
-       (target :clj `(pedestal/spy ~expr)
-               :cljs `(glogi/spy ~expr)))
+     (defmacro spy
+       ([expr]
+        (target :clj `(pedestal/spy ~expr)
+                :cljs `(glogi/spy ~expr)))
+       ([expr & exprs]
+        (target :clj `(last (pedestal/spy [ ~expr ~@exprs]))
```

Comment:
Had to add last here for consistency with ClojureScript. If `spy` were used for production logging, I would wonder  about this implicit vector, but it isn't.